src/webapp/components/survey/SurveyForm.tsx

```diff
@@ -81,10 +88,23 @@ export const SurveyForm: React.FC<SurveyFormProps> = props => {
 
     const updateQuestion = (question: Question, stageId?: string) => {
```

Comment:
You already did in https://github.com/EyeSeeTea/amr-surveys/pull/25 : Move to the hook

Reply:
will be fixed on merge. PR25 yet to be merged.

Reply:
done.